python/ray/autoscaler/_private/aws/node_provider.py

```diff
@@ -420,13 +420,15 @@ def _create_node(self, node_config, tags, count):
                     cli_logger.print(
                         "create_instances: Attempt failed with {}, retrying.",
                         exc)
-        return created_nodes_dict
 
         # TODO: Idempotently correct CloudWatch setup errors on cached nodes?
         node_ids = [n.id for n in created]
-        CloudwatchHelper(self.provider_config, node_ids, self.cluster_name).\
+        CloudwatchHelper(self.provider_config, node_ids, self.cluster_name). \
             setup_from_config()
 
+        return created_nodes_dict
```

Comment:
Did the code just not run before, moving return makes it actually happen?

Reply:
Right - this was a side-effect of a bad merge.